docs/sdk/client.md

```diff
@@ -20,103 +20,95 @@ yarn add @aragon/sdk-client
 ```ts
 // For local testing
 const contextParams: ContextParams = {
-    network: 31337,
-    signer: new Wallet("privateKey"),
-    daoFactoryAddress: "daoFactoryAddress",
-    web3Providers: ["http://localhost:8545"],
+  network: 31337,
```

Comment:
Seems strange to me that these docs are here, unless I'm missing something?

Reply:
I think this is some auto-generated from @josemarinas work. I didn't add or write this